server/src/util/event-email.ts

```diff
@@ -227,8 +246,32 @@ export const eventCancelationEmail = (event: CancelEvent) => {
   );
 };
 
-export const eventConfirmAttendeeEmail = (eventName: string) =>
-  withUnsubscribe(eventConfirmAtendeeText({ eventName }));
+interface ConfirmAttendeeData {
+  name: string;
+  streaming_url: string | null;
+  venue: venues | null;
```

Comment:
```suggestion
  venue: { name: string | undefined } | null;
```
`venues` type has few fields that won't be used here at all. `name` is at most what interests us at this point.

Reply:
Thank you for that! I was having trouble in resolver.ts accepting updatedUser.event as a single parameter, which is how I ended up with "venue" stuff and VS Code stopped complaining. I couldn't figure out what was the data type of updatedUser.event. I should've done what I've always done and looked to the interface EventInvite as an example for that venue name. Now resolver.ts can stop complaining.